Engine/src/Engine/Rendering/UIRenderer.cpp

```diff
@@ -236,5 +238,10 @@ namespace Engine
 	{
 		m_hierarchyPanel.addEntity(tag, entity.getHandle());
 	}
+
+
+	glm::vec2 UIRenderer::getGameWindowPos() {
```

Comment:
Bracket on newline - but this can also be moved into the header, if desired. Might be cleaner, but just a suggestion!

Reply:
Moved to header